conanfile.txt

```diff
@@ -28,13 +25,13 @@ boost:without_math=True
 boost:without_mpi=True
 boost:without_program_options=True
 boost:without_python=True
+boost:without_nowide=True
 boost:without_random=True
 boost:without_regex=True
 boost:without_serialization=True
 boost:without_stacktrace=True
-boost:without_system=True
```

Comment:
Seems we removed 3 blocked items here:
```
boost:without_container=True
boost:without_exception=True
boost:without_system=True
```

We also added 1:
```
boost:without_nowide=True
```

What's the reason for the change? Are we running a different version of Boost?

Reply:
The newer version of boost introduced nowide, which we do not use, hence the addition of without_nowide. On the other hand, atomic and chrono now requires system, contain and exception to work.